### PR TITLE
More Consistent Disposal Chute Throwing

### PR DESCRIPTION
**Pull request by Flaborized**
## About the PR 

This PR:
- Changes throwing items and mobs into disposal chutes to always occur, instead of randomly/rarely occurring.
- Changes the check to enable throwing all throwable mobs into chutes. So you can throw Jones into a chute if you really wanted to
- Changes the chance for the chute to be set off when a mob is thrown in from 20% to 10%, because by making the chance a mob will enter the chute at all and therefore trigger the entire sequence of probabilities 100% from 30%, it -actually- increases the chance that throwing a mob into the chute will set it off from 6% to 10%. Should still be pretty rare.
- People with the clumsy mutation, such as clowns, will always set off the chute and be flushed when thrown in. Nice.

Implementation otherwise the same. Does not work on wall mounted chutes or mail chutes, like normal.

## Why's this needed? 

Throwing into chutes is a cool mechanic that takes advantage of the tons of things that throw you. However, you can't really effectively utilize it since it's pretty rare. This is meant to allow people to do more creative stuff by making it so you can *know* it'll work every time. I also personally feel that the chance to do literally nothing at all with no feedback is bad makes players think that the cool mechanic doesn't exist. There are other things that work like this but I think this one's the easiest to adjust and probably the most fun.

Some things enabled by this (that you could already do it's just like you had to be really lucky):

- Throwing trash bags into disposal chutes as janitor.
- Throwing Jones The Cat into a disposal chute
- Using clownshot to blast the clown into a disposal chute, which immediately flushes and sends them away
- Getting on a chair and flipping across a room into a chute to escape danger
- Using literally any source of throw to send whatever mob or item you want into a chute.

## Changelog

```changelog
(u)Flaborized
(+)Throwing things into disposal chutes isn't luck based anymore, and you can throw more mobs into chutes. Clumsy people will always set the chute off when thrown in.
```

### CODE REVIEW

File: code/WorkInProgress/recycling/disposal_chute.dm [hunk 1]
```diff
@@ -196,47 +196,25 @@
...
+			if(prob(10) || H.bioHolder?.HasEffect("clumsy"))
+				src.visible_message("<span class='alert'><B><I>...accidentally hitting the handle!</I></B></span>")
+				H.show_text("<B><I>...accidentally hitting the handle!</I></B>", "red")
+				flush = 1
+				if (!is_processing)
+					SubscribeToProcess()
+					is_processing = 1
+				update()
+			else
+				update()
```
Comment (flrsh) [inline code/WorkInProgress/recycling/disposal_chute.dm:217]:
```suggestion
			update()
```
I know you were just copying the previous code, but yeah.

**Review by flrsh — APPROVED** [review]

File: code/WorkInProgress/recycling/disposal_chute.dm (hunk 1 above)
Comment (Flaborized) [reply code/WorkInProgress/recycling/disposal_chute.dm:217]: Shoot yeah I saw that and thought to fix it but then forget !

**Comment by ThatFiveGuys** [comment]
Okay I very much like this, but have the animation of it teetering on the edge of the chute still be a thing for like a ten percent chance, I think the animation is neat.

**Comment by Tarmunora** [comment]
All kobe, no yeet

**Comment by Momoberry** [comment]
Damn, I'm gonna try and utilize this with the slip trait for clean getaways with my banana as a clown. Seems fun

**Pull request merged**